dev/src/reference.ts

```diff
@@ -1066,6 +1070,7 @@ export class QueryOptions<T> {
 
     options = options.with({
       kindless: true,
+      requireConsistency: true,
```

Comment:
This should be false (and you should pass it in since there is not really a reason why a kindless all-descendants query should not require consistency by default)

Reply:
done.